components/CheckBox/CheckBox.tsx

```diff
@@ -65,15 +71,45 @@ function CheckBox({ filters, handleFilters }) {
     } else {
       newCheck.splice(currentId, 1);
     }
+
     setChecked(newCheck);
     localStorage.setItem("checked", JSON.stringify(newCheck));
     handleFilters(newCheck);
   };
 
-  function isChecked(id) {
-    return !(Checked.indexOf(id) === -1);
+
+  const allChecked = (index: number) => {
```

Comment:
For just maintaining a name convention, for functions that returns a boolean you can use prefixes like 'is', 'has', 'can'. In this case, you can change these name for `isAllChecked` and `isNoneChecked`.

That way, it is easier to others programmers what is his purpose.

Reply:
Done ;)